testdata/dnn/download_models.py

```diff
@@ -907,7 +859,7 @@ def get_confirm_token(response):  # in case of large files
         filename='yolov4x-mish.weights'),
     Model(
         name='GSOC2016-GOTURN',  # https://github.com/opencv/opencv_contrib/issues/941
-        downloader=GDrive('1j4UTqVE4EGaUFiK7a5I_CYX7twO9c5br'),
+        gdrive='1j4UTqVE4EGaUFiK7a5I_CYX7twO9c5br',
```

Comment:
At the current script state extracting this specific part of functionality to separate class hierarchy seemed unnecessary to me. By rewriting it as a function we've reduced code duplication (actual downloading). In the future we can extract every bit of functionality as a separate class (downloaders, hash checkers, etc.) to improve extensibility, but it does not seem to be particularly useful at this moment because script is still very simple.